docs/vtex-io/Storefront-Guides/b2b-setup/querying-b2b-order-statuses.md

```diff
@@ -0,0 +1,293 @@
+---
+title: "Querying B2B order statuses"
+slug: "querying-b2b-order-statuses"
+hidden: false
+createdAt: "2022-07-12T19:09:55.157Z"
+updatedAt: "2024-03-04T19:09:55.157Z"
+---
+
+> ⚠️The Querying B2B order statuses app has been deprecated in favor of the [B2B Orders History](https://developers.vtex.com/docs/apps/vtex.b2b-orders-history) app. Although this app has limited support, we strongly recommend you update your store theme with the [B2B Suite](https://developers.vtex.com/docs/apps/vtex.b2b-suite) app to keep up with the component's evolution.
```

Comment:
Em "Although this app has limited support", não ficou claro qual app tem limited support. B2B Order status ou B2B Orders History?